src/gcodeExport.cpp

```diff
@@ -1726,6 +1726,10 @@ void GCodeExport::finalize(const char* endCode)
     for (int n = 1; n < MAX_EXTRUDERS; n++)
         if (getTotalFilamentUsed(n) > 0)
             spdlog::info("Filament {}: {}", n + 1, int(getTotalFilamentUsed(n)));
+}
```

Comment:
I think you should still flush here as well.

Reply:
3bb6d32 added it here